openpype/hosts/houdini/plugins/publish/extract_vdb_cache.py

```diff
@@ -14,8 +14,12 @@ class ExtractVDBCache(publish.Extractor):
     label = "Extract VDB Cache"
     families = ["vdbcache"]
     hosts = ["houdini"]
+    targets = ["local", "remote"]
 
     def process(self, instance):
+        if instance.data.get("farm"):
```

Comment:
When this triggers on the farm what is the code that is setting `instance.data["farm"] = False`

Reply:
I think it was already implemented in #3225 

https://github.com/ynput/OpenPype/blob/40037b050ce287f503bf4214f63e9e9f1b8028f5/openpype/modules/deadline/plugins/publish/collect_publishable_instances.py#L36